grafana-plugin/.config/webpack/webpack.config.ts

```diff
@@ -46,11 +50,11 @@ const config = async (env): Promise<Configuration> => {
       '@grafana/slate-react',
       'react',
       'react-dom',
+      'react-router-dom',
```

Comment:
This seems to be needed even with the compat version, otherwise Insights won't render due to their dependency on v5.